package.json

```diff
@@ -10,6 +10,7 @@
   },
   "dependencies": {
     "@react-three/fiber": "^8.0.19",
+    "@tanstack/react-query": "^4.24.9",
```

Comment:
Added react-query to make infinite queries easier to use. Would like to migrate fully to react-query later on.